src/SourceBuild/content/Directory.Build.props

```diff
@@ -4,6 +4,7 @@
     <ImportNetSdkFromRepoToolset>false</ImportNetSdkFromRepoToolset>
     <_SuppressSdkImports>true</_SuppressSdkImports>
     <Configuration Condition="$(Configuration) == ''">Release</Configuration>
+    <EnableSourceControlManagerQueries>false</EnableSourceControlManagerQueries>
```

Comment:
Your description made me think this was only needed for the package-source-build project?  Is there a reason to set it here versus the package-source-build project?

Reply:
Yes, it's only needed for package-source-build. But given the intent of the property, it seemed relevant to apply it to all projects in this context to avoid potential issues in the future.

Reply:
>  it seemed relevant to apply it to all projects in this context to avoid potential issues in the future.

That makes sense but keep in mind it won't flow into the project repo build the way it is set so it really has limited value here.